include/TGUI/Widget.hpp

```diff
@@ -428,7 +428,7 @@ namespace tgui
         ///
         /// Examples:
         /// @code
-        /// widget->setUserData("Data");
+        /// widget->setUserData("Data"); ///Warning: const char*
```

Comment:
The warning is a good idea. But maybe something slightly more verbose like `// Note: type to retrieve with getUserData is 'const char*' here`?